package Pack;
/*
public class No55_Exception1
{
	public static void main(String[] args) {
		System.out.println(1);
		try {
			int a = 10 / 0;
		} catch (Exception e) {
			e.printStackTrace();
		}
		System.out.println(2);
		int[]ar = {1, 2, 3};
		// Array Index OutOf Bounds Exception
		try {
			ar[3] = 10;
		} catch (Exception e) {
			
		}		
		System.out.println(3);
		
		String s = "무궁화꽃이피었습니다";
		System.out.println(s.length());
		s = null;
		
		// NullPoiterException
		try {
			System.out.println(s.length());	
		} catch (Exception e) {
			// TODO: handle exception
		}
		System.out.println(4);
	}
}
*/
